spec/support/views/custom_html_size.erb

```diff
@@ -1,13 +1,19 @@
 <html>
-<head>
-  <title>custom_size</title>
-  <style>
-    html {
-      width: 1280px;
-      height: 1024px;
-    }
-  </style>
-</head>
+  <head>
+    <title>custom_size</title>
+    <style>
+      html {
+        width: 100%;
+        height: 100%;
+      }
+
+      .spacer {
+        width:  1272px; /* using scrollWidth  adds 8px */
+        height: 1008px; /* using scrollHeight adds 16px */
```

Comment:
@coffeejunk I think, I got why we need to use `scroll` references over `offset`, but why is it different from `Width` gap, it should be the same, isn't'?
`height: 1016px; /* using scrollHeight adds 8px */`

Reply:
@Mifrill to be quite honest, the change is almost a year old, and I don't remember anymore why it would've been 16px vs. 8px. From what I see right now, I think you are right in that it should be 8px. :/

Reply:
@coffeejunk no problem, yeah, this "gap" by the defalut Chrome margins:
https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/html/resources/html.css;l=57-60